lib/iex/lib/iex/dochelp/erlang_stub.ex

```diff
@@ -0,0 +1,48 @@
+defmodule IEx.DocHelp.ErlangStub do
+  @moduledoc """
+  Stub module to suggest possible addons for Erlang documentation.
+
+  Currently only returns {:unknown, doc_list }
+  with a suggestive error message.
+  """
+
+  @suggestive "and currently there is no helper installed to provide Erlang documentation"
```

Comment:
Perhaps tell the user what command to run or where to look to fix this problem....